tests/compose-e2e/compose_test.go

```diff
@@ -49,15 +49,8 @@ func TestLocalComposeUp(t *testing.T) {
 
 	const projectName = "compose-e2e-demo"
 
-	t.Run("build", func(t *testing.T) {
```

Comment:
Note to reviewer: removed for now, we will add tests for build later

Reply:
OK, if it's really in the way ; could we not keep this until we replace it by more proper test on build ? 
(just I prefer to not reduce e2e coverage, as we don't have too much there ATM)

Reply:
could we just point to the /sentences compose files and make this pass ?

Reply:
The build fails if I point it to the sentences compose file because it can't find anything to build...

I'll create a new compose file with build, later